src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -198,6 +199,19 @@ public static void showMemberAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredMemberList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the member at the given {@code targetIndex} in the
```

Comment:
Javadocs: filtered list to show only the booking at given index*

Reply:
ok corrected